subprojects/core/src/main/java/org/gradle/api/internal/FeaturePreviews.java

```diff
@@ -28,7 +28,8 @@
     public enum Feature {
         GRADLE_METADATA(false),
         GROOVY_COMPILATION_AVOIDANCE(true),
-        ONE_LOCKFILE_PER_PROJECT(true);
+        ONE_LOCKFILE_PER_PROJECT(true),
+        DEPENDENCY_VERSION_SORTING(true);
```

Comment:
:o: I think I would prefer a different name here. It sounds weird: we already have "dependency version sorting". What this is about is a new sort order. So I think I would prefer something around the lines of `VERSION_SORTING_V2`

Reply:
I was thinking of the name in its context. It is `Feature.DEPENDENCY_VERSION_SORTING` really.
But your proposal works as well.

Reply:
Thinking some more about it, I do not like the `V2` part. There will be no `V3` 😉 
What about `UPDATED_VERSION_SORTING`?

Reply:
I'm just concerned that if in a few months we have _another_ version sorting, then we have difficulties finding good names.